guides/common/modules/proc_manually-tuning-puma-workers-and-threads-count.adoc

```diff
@@ -2,15 +2,11 @@
 = Manually tuning Puma workers and threads count
 
 If you decide not to rely on xref:Puma_Workers_and_Threads_Auto_Tuning_{context}[], you can apply custom numbers for these tunables.
-In the example below we are using 2 workers, 5 and 5 threads:
+In the example below we are using 2 workers and 5 threads:
```

Comment:
```suggestion
The following example uses 2 workers and 5 threads:
```
I know that we are not supposed to attribute actions to non-humans, but "we" or "you" looks odd.

Reply:
The "we" style is used almost everywhere in the guide so I'd prefer to keep that out of scope.

Reply:
Fine with me. I created a bug to update this guide to comply with our guidelines so that should catch it.